python/ccxt/async_support/base/exchange.py

```diff
@@ -393,7 +393,7 @@ async def fetch_market_leverage_tiers(self, symbol, params={}):
         if self.has['fetchLeverageTiers']:
             market = await self.market(symbol)
             if (not market['contract']):
-                raise BadRequest(self.id + ' fetch_leverage_tiers() supports contract markets only')
+                raise BadRequest(self.id + ' fetch_market_leverage_tiers() can not be emulated, because fetch_leverage_tiers() supports contract markets only')
```

Comment:
I suggest that we keep these error messages short as they were, without mentioning emulation.